modeling-cmds/src/ok_response.rs

```diff
@@ -22,6 +22,13 @@ define_ok_modeling_cmd_response_enum! {
             units,
         };
 
+        /// The response of the `EngineUtilEvaluatePath` endpoint
+        #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, ModelingCmdOutput)]
+        pub struct EngineUtilEvaluatePath {
+            /// The evaluated position
```

Comment:
Which position along the path is this?

Reply:
the evaluated one from the t value passed in.